virttest/qemu_devices/qdevices.py

```diff
@@ -1489,15 +1489,17 @@ class QIOThread(QObject):
     ["id", "poll-max-ns", "poll-grow", "poll-shrink"]
     """
 
-    def __init__(self, iothread_id, params=None):
+    def __init__(self, iothread_id, params=None, vq_bus=False):
```

Comment:
it looks that vq_bus is never used here

Reply:
vq_bus was removed from another commit, when I review all code change, it was gone, while when I review this commit 'qdevices: add QIOThreadVQbus' it still exists.  it may be better to keep this commit clear because it looks an independent commit